src/main.rs

```diff
@@ -52,6 +52,11 @@ fn main() {
             return;
         }
     };
+
+    if { true } {
```

Comment:
This changeset will be removed once we know travis shows:
* an error in [this job](https://travis-ci.org/fxbox/foxbox/builds/106964559)
* but no in [this one](https://travis-ci.org/fxbox/foxbox/builds/106964277)